api-spec/v2.1/transactions-api.yaml

```diff
@@ -278,6 +278,7 @@ components:
         - IO
         - CHECKOUT
         - CHECKOUT_CART
+        - WISP
```

Comment:
In order to avoid breaking changes we had decided not to add clientId or am I wrong?

Reply:
This is used only in `POST /transactions`, so it's not breaking
It's still needed as a change to support the use case of activating a transaction without calling verify on it beforehand